tests/inside_worker_test/conftest.py

```diff
@@ -85,6 +85,7 @@ def osmaxx_functions(osm_tables):
         'sql/functions/0030_transliterate.sql',
         'sql/functions/0040_interpolate_addresses.sql',
         'sql/functions/0050_cast_to_int.sql',
+        'sql/functions/0060_cast_to_float_or_null.sql',
```

Comment:
@das-g is there a reason we keep [two](https://github.com/geometalab/osmaxx/pull/648/files#diff-6b9af5ce056da868f1e31805af76335cR201) lists of these sql scripts in the tests?